### ISSUE TITLE
Review annotations using extension 'exists_during' 

### ISSUE THREAD

**Issue by pgaudet** [geneontology/go-annotation#2677]
Dear all, 

We plan to remove the GO_REL extension 'exists_during'. There are a lot of annotations to this term. The suggestion is to change the annotations from 
`CC exists_during BP`
to `BP occurs_in CC` 
ie make sure the gene product is annotated to the process. 
If annotations are to phases (ie cell cycle stages), then you can annotate `MF happens_during BP occurs_in CC`. 

https://docs.google.com/spreadsheets/d/1ba0z4BtMJLG4Z0QuS1mhl60TZrdfU719nZr3Ig5uQaQ/edit#gid=0

Database | Count
-- | --
ComplexPortal | 7
dictyBase | 165
MGI | 109
PomBase | 1678
RNAcentral | 6
SGD | 305
UniProtKB | 392
WB | 127

Thanks, Pascale

**Comment by ukemi** [comment]
This suggestion wouldn't be correct. All CC exists during means is that the component exists while the process is happening. It doesn't say anything about the process happening in the component.

**Comment by pgaudet** [comment]
Well, looking at the annotations I am not sure this is the case. 

For example: 

Creb1 | cAMP responsive element binding protein 1 |   | ATF4-CREB1 transcription factor complex | exists_during response to endoplasmic reticulum stress
-- | -- | -- | -- | --

The paper (PMID:12871976) states: "Here we have identified the closely related ATF1 and CREB1 as nuclear co-factors that form in vivo complexes with endogenous ATF4. ER stress induces CREB1 phosphorylation and ATF1/CREB1 binding to the Grp78 promoter. "

So, I dont think the complex forms during ER stress, but it is active during ER stress. I think the guidelines are to annotate the function of the complex as 'part of' the BP. Right?

**Comment by vanaukenk** [comment]
I think we meant to suggest that the MF 'occurs in' the CC and to make sure that the gene product is also annotated to the BP.

**Comment by pgaudet** [comment]
OK, so another suggestion would be to replace the annotation by 'located_in' ?

**Comment by pfey03** [comment]
For me also problematic, and only looked at one.

glkA part_of leading edge exists_during chemotaxis to cAMP

I now changed it to: 
glkA part_of leading edge part_of chemotaxis to cAMP but that's the only other relation but isn't right as well

**Comment by vanaukenk** [comment]
David, Pascale, and I are going to discuss this further; for now, curators can hold off on making any changes.

**Comment by vanaukenk** [comment]
Thanks for the rapid feedback, @pfey03 
It always helps to have concrete examples to discuss!

**Comment by bmeldal** [comment]
The 7 CP annotations are probably from Ruth's group as we don't add extensions but I recognise the complex as one @BarbaraCzub worked on.

My 2p:
`CC exists_during BP` suggests to me a coincidental relationship while `CC involved_in BP` suggest to me that the CC is actually having an effect on the BP. Not sure `CC part_of BP` makes much sense - but I'm probably wrong here ;-)

**Comment by vanaukenk** [comment]
A CC cannot be 'part of' a BP.

**Comment by RLovering** [comment]
Hi 
the complex annotations are UCLs. I have updated the table above.
An example of one of our annotations
C1qa |   | GO:0045202 | synapse 
|exists_during(UBERON:0007221, neonate),part_of(EMAPA:35290, dorsal lateral
  geniculate nucleus)\
|exists_during(UBERON:0007221),part_of(EMAPA:35741, retina inner plexiform
  layer)
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --

**Comment by ValWood** [comment]
Most of our annotations are

CC exists during 'phase'

Biological phases are in the process ontology, but they are not processes, they are periods  when things occur. Ideally they would be  in an orthogonal ontology, and they are not for direct annotation.

Here is an example I am currently:
<img width="424" alt="Screenshot 2019-10-18 at 07 28 17" src="https://user-images.githubusercontent.com/7359272/67071631-f29a1880-f17a-11e9-8664-f25d78b7b5ef.png">

for3 localizes to the cell tip during 'mitotic interphase', to the medial ring during 'mitotic anaphase' and to the septin ring during 'mitotic metaphase'. These are clearly where this gene product is acting  at these times-recording the different location in different phases is really important to our users. Most gene products involved in cell cycle processes relocate multiple times during the cell cycle, and these phas-specific  location pattern are one of the main clues to the biological role.

It clearly wouldn't make sense to invert this in any way.

**Comment by pgaudet** [comment]
Are all your annotations capturing transient localizations ?

**Comment by ValWood** [comment]
These are the locations where these proteins act. They are 'transient' in that they are not permanent  but these most proteins  have different roles at different locations at different  times in the cycle. The cytoskeleton moves around. It's at the cell tips during interphase when it is delivering polarity factors for growth, and at the cytokinetic ring during mitosis.   Sometimes the locaization precedes the complete knowledge about exactly what the protein is doing (in fact it is this information that drives forward the dissection of the process). We have no other way of describing this and it is critical for our users.  We will also incorporate this information into our automated GO-Cams.

We would not curate locations which are thought/known to be transit locations (for example the ER location or Golgi of a cell surface protein).

**Comment by ValWood** [comment]
The use of 'exists_during' has been established for a long time. It wasn't an example  in the annotation extensions publication, but it was one of the  already established relationships discussed  broadly  and approved for use at this time.   We have been using this at PomBase since before AE existed.

**Comment by ValWood** [comment]
Any update on the use of non-phase terms for "exists_during"?

**Comment by pgaudet** [comment]
We still would prefer using 'part_of' for response_to terms.
@vanaukenk

**Comment by ValWood** [comment]
We don't have many response_to and I would be happy to lose those.

However we have things like

during conjugation, quiesence, karyogamy.

These are useful  to our users, especially for proteins that are not well studied, to know the locations are during these processes, but not enough information to annotate to the process. We don't know what to do with these but we would like to keep them, because we know that people use them. These types of annotation are referenced sometimes as providing  functional clues. I guess we could filter them out of the GO submission.

**Comment by vanaukenk** [comment]
I've submitted a ticket to the RO tracker to see if we can use 'existence overlaps' from RO to replace 'exists during' in GO:
https://github.com/oborel/obo-relations/issues/349

**Comment by RLovering** [comment]
Hi 

I agree with David's comments above and UCL applied the relation assuming that: CC exists during means is that the component exists while the process is happening. It doesn't say anything about the process happening in the component.
I have just looked at the ARUK annotations and these definately should not be replaced with involved in the BP. I have deleted these exists during statements but I cannot do this for >200 UCL annotations. I would be happy for all of the UCL exists during annotation extensions just to be removed

Ruth

**Comment by ukemi** [comment]
I just checked this morning and I don't see any of these annotations in the MGI GO-CAM import file that we provide to the GOC. I must have cleaned them all up at some point.

**Comment by vanaukenk** [comment]
Thanks @RLovering and @ukemi 
I've put this on the agenda for Monday's ontology call.  I'm hoping we can get the ontologists to confirm the replacement of the GOREL with the RO relation, existence_overlaps.
I'll post the outcome here.
At WB we used the relation the same way as UCL.

**Comment by vanaukenk** [comment]
Reviewing the WB annotations that used 'exists during', I could replace that annotation extension relation with 'existence starts and ends during' (RO:0002491) but would need to have that relation available in Protein2GO.

@alexsign @pgaudet How should we move forward with this?

**Comment by pfey03** [comment]
Yes, existence starts and ends during' (RO:0002491) might also be too specific to just swap our many relations, although that term has the exact synonym 'exists during'. I would need to inspect some but need some time to do that as I have other urgent things on my plate.

existence overlaps seems also not ideal for us.

**Comment by vanaukenk** [comment]
Thanks for your thoughts @pfey03 @mah11 

I've been systematically reviewing the WB annotations and have found that we used 'exists during' in slightly different ways in conventional annotations, so we'll not just be swapping out all of our current 'exists during' (GO_REL:0000032) for 'existence starts and ends during' (RO:0002491).

In some cases, I am actually deleting the 'exists during' extension, as I can see that its previous use is better suited to capturing where an activity (MF) occurs as part of a BP in a GO-CAM model.

In other cases, such as cell cycle phases or C. elegans developmental stages, I can more confidently replace the AE relation with 'existence starts and ends during' (RO:0002491) but will need it added to Protein2GO to do so, as I am planning to make these changes manually as we don't have that many.

We could ask @alexsign to add 'existence starts and ends during' (RO:0002491) and 'existence overlaps' (RO:0002490) to Protein2GO, as that would help curators make any necessary updates while they're reviewing the annotations, but at least for the WB set, I don't think we could have made blanket changes, as some assertions would then have been incorrect.

**Comment by pfey03** [comment]
Thanks @vanaukenk, we have so many, it will be quite a project to move them to correct relation, maybe some can even become part_of. Please ask Alex to keep the old term for an overlap of several weeks?
Thanks

**Comment by mah11** [comment]
Thanks from me too, @vanaukenk. That brings me to a very simple question: Will 'existence overlaps' be allowed in extensions in our submitted GAF & GPAD files?

**Comment by vanaukenk** [comment]
@mah11 

At present, I don't see why not.  For capturing this kind of information, we may well need to use more or less specific relations depending upon what is known.

One of the main objectives with the AE relation review, as I understand it, is to converge on RO as a single source for all of the relations used throughout GO even if that means adding new relations to RO to capture what curators were otherwise trying to say with a GO_REL relation.

**Comment by mah11** [comment]
@vanaukenk OK, thanks again. At some point in the not-too-distant future, we'll replace exists_during with 'existence overlaps'. If there's a deadline for exists_during to disappear, we'll meet it; we may get to it sooner depending on when it is and what else we have on.

**Comment by pfey03** [comment]
I have a deadline June 30, so I'll get to it in July

**Comment by RLovering** [comment]
Hi

I think that there is a RO file that Protein2GO uses to provide the AE relation options. I am not sure who is responsible now for maintaining that file - does anyone know? I think that it might be this file here: https://github.com/geneontology/annotation_extensions/blob/master/display_for_annotators.tsv although there is a typo in the first line of this file so maybe this is not the one that is used.

If this is the file then from Kimberley's statements above it sounds like the file needs to be updated with:

obo id / shorthand | label | RO/GOREL ID | subsets
-- | -- | -- | --
existence_overlaps | existence overlaps | RO:0002490 | AE_biological_process, AE_developmental_stages, display_for_curators
existence_starts_and_ends_during | existence starts and ends during | RO:0002491 | AE_biological_process, AE_developmental_stages, display_for_curators

While this file is being edited are there any other relations should be removed? 

I am not sure if this file is then used to create the AE relation ontology graph https://www.ebi.ac.uk/QuickGO/annotationExtensionRelations that is linked from Protein2GO. As ideally this would be updated too.

@alexsign please confirm the location of the source files for both the Protein2GO AE relation options and also the AE relation graph

Thanks

Ruth

**Comment by mah11** [comment]
I've just committed a PomBase GAF using `existence_overlaps` where we previously had `exists_during`. Hope it all goes well!

**Comment by pfey03** [comment]
I have some CC annotations exist during mitotic phase. I see the relation coincident_with, is this a valid relation? For now I put part_of but not too happy with this; existence_overlaps is not in P2GO at least not for CC.

All our exists_during are to CC terms.

**Comment by RLovering** [comment]
Hi Susan Clew at HGNC need to address their annotation but I don't know the github handle for HGNC annotations that are not UCL HGNC annotations

**Comment by RLovering** [comment]
Sorry I am still confused. I have a lot of annotations in this format:
synapse exists during UBERON_0007221  neonate stage

so should these change to
synapse existence_overlaps UBERON_0007221  neonate stage

and then for 
ATF4-CREB1 transcription factor complex  exists during GO:0034976 response to endoplasmic reticulum stress

These need to change to
ATF4-CREB1 transcription factor complex existence_overlaps GO:0034976 response to endoplasmic reticulum stress

Confirmation would be appreciated

Thanks

Ruth

**Comment by mah11** [comment]
Well, we now get a slew of syntax errors for all the extensions that I "upgraded" to existence_overlaps(GO:nnnnnnn), so ...

I'd say the existence_overlaps examples @RLovering has (previous comment) matches what we're planning to do, and indeed tried doing in the last GAF, but it isn't working yet. @vanaukenk do you think it's worth switching back to exists_during temporarily?

We're not going to change any PomBase CC exists_during(GO:nnnnnnn) to BP occurs_in CC no matter what happens with existence_overlaps, though -- they do. not. mean. the. same. thing. and if we had data to annotate to a BP we'd do it instead of or in addition to a CC annotation. Furthermore, most of our during(GO) annotations are during cell cycle phases or response-to-stimulus terms that we wouldn't use for direct BP annotation.

**Comment by ValWood** [comment]
Yes,  everyone please remember that the branch of GO which refers to phases should not really be in the BP ontology at all. These are not Biological processes,  they represent temporal aspects of processes  and their placement in the BP ontology is a fudge.  Use of these terms for direct annotation is prohibited (as it should be because they aren't biological processes).

If this causes confusion, maybe these phase terms should move into their own ontology aspect. This was discussed in the past  during the cell cycle overhaul when these terms were collected into this distinct  branch and disambiguate from related processes (i.e. S-phase/DNA replication, M-phase and mitosis etc, etc).

 @cmungall  thoughts?

**Comment by vanaukenk** [comment]
@mah11 - I've put this on the agenda for the 2021-07-20 annotation call.

**Comment by vanaukenk** [comment]
@mah11 
Don't revert your PomBase file just yet.
I'm working with @dustine32 and @kltm to get 'existence overlaps' into the file that checks for valid annotation extension relations:  https://github.com/biolink/ontobio/pull/580

**Comment by ValWood** [comment]
Is this a hard check or a soft check? If it is a hard check, I would rather we revert to 'exists_during' until resolved, as a hard check  loses nearly 2000 PomBase annotations from the GO database. Even if only temporary, if 3rd parties pick up the annotation with this chunk missing this could affect end users  GO analysis for a long time.

**Comment by vanaukenk** [comment]
For updating annotation extension relations in Protein2GO, gorel-edit.owl was updated according to these two PRs:
https://github.com/geneontology/go-ontology/pull/21961
https://github.com/geneontology/go-ontology/pull/21962

**Comment by pfey03** [comment]
still not available in P2GO Will check often now and start on updating our CC annotations when available

**Comment by vanaukenk** [comment]
@pfey03 
We just made the updates to the gorel-edit.owl file this morning.  It may take a day or so to trickle down to Protein2GO.

**Comment by pfey03** [comment]
ok thanks @vanaukenk

**Comment by pfey03** [comment]
Dicty annotations are done. 
Will check after next P2GO release in case there is any stray annotation left

**Comment by Antonialock** [comment]
the original message still says "NO ACTION NEEDED YET FROM CURATORS"? - is this still true?

**Comment by vanaukenk** [comment]
@Antonialock 
No; I've removed the message.
The replacement relations for 'exists during', i.e. 'existence overlaps' and 'existence starts and ends during', are now available in Protein2GO.  http://wiki.geneontology.org/index.php/Annotation_Conf._Call_2021-09-07
Please let me know if you have any questions about updating annotations.

**Comment by RLovering** [comment]
Hi Antonia 
you might find this link useful https://www.ebi.ac.uk/QuickGO/annotations?extension=exists_during

I was hoping to ask Alex to automatically change these to existence overlaps as there is no way I am editing 220 annotations manually - if I don't need to

What do you think?

Ruth

**Comment by Antonialock** [comment]
@RLovering that sounds very sensible! 
@alexsign

**Comment by RLovering** [comment]
Hi Alex

please would you change all the UCL (HGNU, PARL, ARUK, BHFL) CC annotation extension relations from exists during to existence overlaps. 

And based on Antonia's comment above please do this for the SWIE annotations too ;)

Please confirm when this is complete so that I can take my name off this ticket.

Thanks and have a good weekend

Best

Ruth

**Comment by alexsign** [comment]
Updated for HGNU, PARL, ARUK, BHFL and SWIE sources. Bellow is the stats for exists_during still in GOA  

SGD	420
RGD	328
AgBase	58
Automated Function Prediction SIG CAFA Experiment (UniProt Curators)	32
FlyBase	6
GO_Central	5
Alzheimers Project at University of Toronto 	4
UniProt (SIB Curators)	2
DictyBase	2
WormBase	2
PHI-base	2
SynGO	1

**Comment by RLovering** [comment]
wonderful, thank you so much Alex
I have updated the Alzheimers Project at University of Toronto annotations too
Ruth

**Comment by pfey03** [comment]
@alexsign for Dicty by dictybase there are none left.  I ran the report for exists_during in P2GO.  Could it be two external annotations? Can you send the IDs?   - Thanks

**Comment by alexsign** [comment]
@pfey03 
2113793648	DDB	protein	Q54LY5	located_in	GO:0000785	ECO:0000314	PMID	21807880
2114261890	DDB	protein	Q54MA9	located_in	GO:0000785	ECO:0000305	PMID	21807880

**Comment by pfey03** [comment]
@alexsign thank you!! I think they escaped my dictybase report because Kimberly touched them last for GP to GO relation

**Comment by vanaukenk** [comment]
@srengel the replacement relations are available in the graph editor; I will ask @tmushayahama to also add them to the CC parts of the noctua form.
